clouddriver-openstack/src/test/groovy/com/netflix/spinnaker/clouddriver/openstack/provider/view/OpenstackLoadBalancerProviderSpec.groovy

```diff
@@ -277,44 +277,7 @@ class OpenstackLoadBalancerProviderSpec extends Specification {
 
   }
 
-  def 'search for all load balancers'() {
```

Comment:
because the aws version in which i used to mimic this functionality did not have those tests